dependencies.yaml

```diff
@@ -3,7 +3,7 @@ files:
   all:
     output: conda
     matrix:
-      cuda: ["11.8", "12.0"]
+      cuda: ["11.8", "12.0", "12.2"]
```

Comment:
I think it might be okay to remove 12.0 here, or make no change (not add 12.2). We don't want to commit a ton of environment files to the repo, I think one per CUDA major version is enough. However, be aware that some repos have README or CONTRIBUTING files that reference this by name, so remember to grep for that and update `all_cuda-120...` to `all_cuda-122` if needed.

Reply:
Got it, makes sense!

Just pushed https://github.com/rapidsai/rmm/pull/1419/commits/4111b4bebfe15d7d8fa7dbbaa21065056cfad7ec removing the 12.0 file from `conda/environments`.

I didn't find any references in docs or elsewhere that needed to be updated.

```shell
git grep 120
```

Reply:
I don't think this pattern occurs in rmm but I think it does in cudf, raft, cuml, and possibly others.

Reply:
Yep yep, I did stuff like this in all the other PRs:

```shell
git grep 120
git grep _cuda-
git grep cuda_all
```

And found a few. Lots of projects are doing that but only with their 11.8 environments.

Here's an example of one that had to be updated: https://github.com/rapidsai/raft/pull/2092/files#diff-45bce2a7a53bb70b279563519336e157236c793fc6448d11e90b25d3a9355691